src/app/gui-elements-module/buttons/cpe-button/cpe-button.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CpeButtonComponent } from './cpe-button.component';
+
+describe('CpeButtonComponent', () => {
+  let component: CpeButtonComponent;
+  let fixture: ComponentFixture<CpeButtonComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ CpeButtonComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(CpeButtonComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
I know this is the default generated test file for a component but should we write tests for a component such as this? I guess we could write tests to confirm the EventEmitter works.

Reply:
Yes, but eventually it will be part of an integration or e2e test anyways. We'll see.